### DISCUSSION TITLE
怎么修改默认语言，现在是中文

### DISCUSSION THREAD

**Discussion by tkfanswind** [spencerwooo/onedrive-vercel-index#411] [Q&A] [answered]
修改默认语言是英文

**Comment by spencerwooo** [comment, marked as answer]
https://github.com/spencerwooo/onedrive-vercel-index/blob/3a0f33630113bbe6ebadf548a5a02d3899d7b878/next-i18next.config.js#L5

**Reply by spencerwooo** [reply]
以及：https://nextjs.org/docs/advanced-features/i18n-routing#automatic-locale-detection
